Note taking and decision making (#35)
carpentries-incubator/meetings-rse: episodes/making_decisions.md

:::::: questions

* How can I make sure meetings end with important decisions having been made?

> * How do I harness participant's energy and opinions into concrete decision making?

Reviewer: ```suggestion
* How do I harness participants' energy and opinions into concrete decision making?
```

If none of the items on your agenda require decision making from the group, you should reconsider whether it is useful to have the meeting at all. 
As discussed previously, information dissemination is usually not an appropriate reason to hold a meeting.
Information can be circulated by email instead, without interrupting peoples' work. 

Given that there are agenda items requiring decision-making, you should be clear about what the role of the meeting participants is in striking a decision.
We go into depth in how to establish decision making roles and responsibilities in another course in this series (Collaboration on Remote Teams). 
In brief, items placed on the agenda should be there because the participants at this meeting have some level of decision making role in their resolution. 
That might be as an advisory body, or as an official body whose decisions are binding.
The individuals at this meeting were invited because you believe that this group of people are likely to have useful insight and expertise in helping make these decisions.

> They may also be included because they will be impacted by the decision, and therefore have a stake in carrying it out.

Reviewer: ```suggestion
They may also be included because they will be impacted by the decision, and therefore have an interest in carrying it out.
```

If we want to avoid "stake" altogether

The difficulty of coming to consensus seems to raise exponentially with the number of meeting participants - so how can you ensure that decisions that need to be made actually get made and not deferred to a future meeting? 

> We recommend that meetings adopt a modified version of [Martha's Rules](https://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1825&context=sociologyfacpub), which are designed to allow groups to make progress without requiring strict consensus. 

Reviewer: In the past I have presented Martha's Rules as a good framework to initiate a discussion about the appropriate decision-making process for a participant's project(s). It is a good system but will often (usually?) feel too rigid and formalised. But even if participants don't want to go the whole way to adopting MR, the Rules highlight some important aspects that they can bring to their projects/meetings:

1. The person making a proposal is responsible for presenting it in a way that the other project members can understand and see how it relates to the project as a whole
2. If somebody objects to the proposal, they must articulate why.
3. If only a minority of the group's members feel strongly about the matter one way or the other, it allows for that minority to discuss the proposal further without taking up the time of the rest of the group.

When we included [a section on governance/decision-making in CLDT](https://github.com/carpentries/lesson-development-training/blob/f88ba06c74b73e871965f6c7d2a5b95bed6076da/episodes/project-management.md?plain=1), we mostly just wanted participants to appreciate the importance of having some documented system for making decisions. Even if they hope to achieve consensus every time, it can be a good idea to have some mechanism to unblock the process e.g. to prevent things from moving very slowly/to stop one person from blocking all progress (the two main hazards of consensus IME).

Perhaps we can rephrase this part to something like

> We recommend that you adopt some form of documented process for decision-making that you can apply consistently in your meetings. 
> Different systems will work more or less well for different groups and different projects, but the important thing is to have one!
> When considering which process might work well for your team, Martha's Rules could be a good starting point.
> Martha's Rules provide a way for groups to make progress without requiring strict consensus.

For instance, participants will be more willing to listen to others if they know that their right to share their opinion is being safeguarded by the Facilitator. 
Martha's Rules also call for agenda items to be set and shared in advance of the meeting, and for each to have an associated time limit. 
You're already well prepared for those elements!

## The Steps

There are five steps to Martha's rules, although the designers note that these five steps do not need to be followed exactly, and should be modified based on the group's needs. The five steps are: 

1) Preparation, 
2) Generating proposals, 
3) Making proposals concrete, 
4) Taking a "sense" vote, and 
5) Taking a "vote" vote.

We've already covered the preparation steps, so let's talk about generating proposals.  

:::::: challenge

## Evaluating Multiple Proposals

Exercise about implementing Martha's Rules - could we use the table from the publication and ask the group which proposals should move forward to the next stage?

:::::: solution

Solution text here.

> :::::::::::::::::::::::::::::::::::: 

Reviewer: Instead of an exercise on implementing MR, could we make this a discussion of the advantages and disadvantages of different strategies, which kinds of projects they could be suited to, etc? With a prompt at the end for the participant to reflect on what kind of system would work well for their project/team?
